portfolio/src/main/webapp/script.js

```diff
@@ -40,9 +40,11 @@ function getMessagesFromServer() {
   });
 }
 
-function createListElement(text) {
+function createListElement(email, text) {
```

Comment:
Hmm, I believe we talked about calling these kind of functions "render" something, what happened with that?

Reply:
This approach is in a later branch.